### PR TITLE
My HTML Portfolio Files

### PR DESCRIPTION
**Pull request by Chigoz169**
I created My Portfolio page using just HTML codes and the Portfolio consists of 4 different pages namely;
index, contact, projects and profile respectively.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About</title>
+    <link rel="stylesheet" href="style_abt.css" />
+    <script
```
Comment (idayatOlaide) [inline about.html]: Only index.html is expected

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Contact</title>
+    <link rel="stylesheet" href="style_ctc.css" />
```
Comment (idayatOlaide) [inline contact.html]: Only index.html is expected

File: index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home</title>
```
Comment (idayatOlaide) [inline index.html]: Consider using your name

File: index.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    <link rel="stylesheet" href="style.css" />
```
Comment (idayatOlaide) [inline index.html]: HTML only. CSS is not expected at this point.
Also, add a favicon, meta tags for title, description, keywords for accessibility.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    <script
+      src="https://kit.fontawesome.com/a126fa987a.js"
+      crossorigin="anonymous"
+    ></script>
```
Comment (idayatOlaide) [inline index.html]: At this stage, <script/> is not meant to be seen within this repository

File: index.html [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+          />
+          <p id="about">I am Abraham Judah and a Nigerian</p>
+          <h2>Frontend Web Developer</h2>
+        </div>
+      </main>
+      <footer>
+        <div id="footer">
+          <section class="link">
+            <h3>Want to connect with me?</h3>
+          <p>
```
Comment (idayatOlaide) [inline index.html]: Don't think we can have section inside a footer.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+            open to projects and collaborations
+          </p>
+          </section>
+          <section class="link">
+            <a href="#"><i class="fab fa-instagram fa-lg" ></i>Instagram</a>
+            <a href="#"><i class="far fa-envelope fa-lg"></i>Email</a>
+            <a href="#"><i class="fab fa-linkedin-in fa-lg"></i>LinkedIn</a>
+            <a href="#"><i class="fab fa-facebook-f fa-lg"></i>Facebook</a>
+            <a href="#"><i class="fab fa-github fa-lg"></i>Github</a>
+            <a href="#"><i class="fab fa-twitter fa-lg"></i>Twitter</a>
```
Comment (idayatOlaide) [inline index.html]: Add the url since it leads no where

File: portfolio.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
```
Comment (idayatOlaide) [inline portfolio.html]: Only index.html is expected

**Review by idayatOlaide — CHANGES_REQUESTED** [review]

File: index.html [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/a126fa987a.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <img src="images/ABJ.png" alt="Logo" width="100px" height="40px" />
```
Comment (chiamakaikeanyi) [inline index.html]: Use a descriptive alt text here and in other occurrences. A sample alt text can be alt="A black and white panda seating on a green grass and holding a green stick into his mouth. It describes the image to blind people and also useful when the image is not loaded. It helps in making apps accessible.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+        <h1>ABRAHAM JUDAH</h1>
+        <nav>
+          <ul>
+            <li><a href="index.html">Home</a></li>
```
Comment (chiamakaikeanyi) [inline index.html]: Modify the code to include the information on other pages on this page. Then, using bookmark, navigate to the various sections. Refer to https://www.w3schools.com/html/html_links_bookmarks.asp

File: index.html [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+        </nav>
+      </header>
+      <main>
+        <div id="profile">
+          <img src="images/picture.jpg" alt="Picture" height="300px"/>
+        </div>
+        <div id="description">
+          <p id="greeting">Hello there!</p>
+          <img
+            src="https://img.icons8.com/emoji/48/000000/waving-hand-emoji.png"
```
Comment (chiamakaikeanyi) [inline index.html]: Add an alternative text

File: index.html [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+        <div id="footer">
+          <section class="link">
+            <h3>Want to connect with me?</h3>
+          <p>
+            You can simply use any of the mediums to reach me as I am currently
+            open to projects and collaborations
+          </p>
+          </section>
+          <section class="link">
+            <a href="#"><i class="fab fa-instagram fa-lg" ></i>Instagram</a>
```
Comment (chiamakaikeanyi) [inline index.html]: Use target="_blank" and add the appropriate URLs.
Note that when you use target="_blank", always add rel="noopener" or rel="noreferrer". Visit https://developers.google.com/web/tools/lighthouse/audits/noopener for more information.

**Pull request merged**